app/scripts/templates/support/create_secure_password.mustache

```diff
@@ -0,0 +1,49 @@
+<div id="main-content" class="panel panel-left-aligned">
+  <header>
+    <h1 id="fxa-support-create-secure-password">{{#t}}Create secure passwords to keep your identity safe{{/t}}</h1>
+  </header>
+
+  <p>Choosing more secure passwords will help keep your identity safe on the internet.<br/>
```

Comment:
Using a `<br />` outside of emails to break lines has always made me feel a little gross. Can we just use two `p` or `div`s?